appcues/src/main/java/com/appcues/statemachine/StateMachine.kt

```diff
@@ -210,7 +210,7 @@ internal class StateMachine(
 
     fun stop() {
         appcuesCoroutineScope.launch {
-            handleAction(EndExperience(markComplete = false, destroyed = true))
+            handleAction(EndExperience(markComplete = false, destroyed = false))
```

Comment:
this is what I mean, when stop is called I think its intentional and we could handle it as destroyed = false, so it can be used in different places, but let me know if this looks wrong we can revert this piece and leave as is.

Reply:
I will leave as it was, we can revisit this later.